tests/utils.go

```diff
@@ -3787,3 +3787,31 @@ func StartVMAndExpectRunning(virtClient kubecli.KubevirtClient, vm *v1.VirtualMa
 
 	return updatedVM
 }
+
+func SetNodeUnschedulable(nodeName string, virtCli kubecli.KubevirtClient) {
+	Eventually(func() error {
```

Comment:
Let's take the opportunity and use the patch here to reduce the possible conflict.

Reply:
Also, there is now `libnode` package where this suits more. Let's not feed the utils anymore...

Reply:
Done, thanks!